yt/visualization/volume_rendering/render_source.py

```diff
@@ -901,7 +901,7 @@ def annotate_mesh_lines(self, color=None, alpha=1.0):
         if color is None:
             color = np.array([0, 0, 0, alpha])
 
-        locs = [self.sampler.amesh_lines == 1]
+        locs = (self.sampler.amesh_lines == 1,)
```

Comment:
I am confused as to why
```suggestion
        locs = self.sampler.amesh_lines == 1
```
doesn't work. Are you sure the issue isn't on the doc build's side?

Reply:
I'm confused as well but I'm not confident that those _should_ be equivalent as far as numpy slicing is concerned.
I'm not sure of anything but the current state seems to do the job and at least it's pretty obvious that it stays faithful to the original code.